airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/errors/messages/ErrorMessage.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.errors.messages;
+
+import io.airbyte.integrations.base.errors.utils.ConnectionErrorType;
+import io.airbyte.integrations.base.errors.utils.ConnectorName;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class ErrorMessage {
+
+  private static final String DEFAULT_ERROR_MESSAGE = "Standard error code: %s. \n Error Message: %s.";
+  protected final Map<String, ConnectionErrorType> CONSTANTS = new HashMap<>();
```

Comment:
The name of `CONSTANTS` does not seem descriptive to tell from a quick glance what this means. It seem that this is actually a mapping of `SQLSTATE` error codes/strings to an ENUM of `ErrorTypes`, correct me if I'm wrong? 

If so, can we have the name of `ERRORCODES_TYPES` to help better identify this constant?

Reply:
renamed to ERRORCODES_TYPES